misk/src/test/kotlin/misk/web/interceptors/TracingInterceptorTest.kt

```diff
@@ -92,11 +92,15 @@ class TracingInterceptorTest {
     val chain = RealNetworkChain(TracingTestAction::call.asAction(DispatchMechanism.GET),
         tracingTestAction, httpCall, listOf(object : NetworkInterceptor {
       override fun intercept(chain: NetworkChain) {
-        tracer.buildSpan("parent-span-exists")
-            .startActive(true).use {
-              tracer.activeSpan().setBaggageItem("hello", "world")
-              chain.proceed(chain.httpCall)
-            }
+        val span = tracer.buildSpan("parent-span-exists").start()
+        val scope = tracer.scopeManager().activate(span)
+        try {
+          tracer.activeSpan().setBaggageItem("hello", "world")
+          chain.proceed(chain.httpCall)
+        } finally {
+          scope.close()
+          span.finish()
```

Comment:
FYI @mpawliszyn this didn’t remain as nice as it was originally, thanks to the need to finish()